docs/administration/configuration/configuring-access/index.md

```diff
@@ -0,0 +1,10 @@
+# Configuring user access
+
+By default confirmation of a user’s email address on sign up is enabled.
+The user is unable to sign in until they confirm their email address.
```

Comment:
Language observations:

- "user" is singular, "they" is plural;
- "sign up" is verb, "sign-up" is noun.

 What about rewording to something like:

```
# Configuring user access

The default REANA configuration allows users to sign up, after which
an email is sent to the user's email address for confirmation.  Once
the email address is confirmed, the user can then ask administrators
for an access token.

If you would like to disable the email verification step, you can add 
`REANA_USER_CONFIRMATION: false` Helm value to [`components.reana_server.environment`](https://github.com/reanahub/reana/tree/master/helm/reana).

If you would like to disable the sign-up form completely, and add your
users manually, you can configure
[`hide_signup`](https://github.com/reanahub/reana/tree/master/helm/reana)
Helm value accordingly.

Note that it is also possible to deploy a Single-Sign On system for
managing user access. Currently, this is only available for CERN
deployments.
```